store/tikv/region_request.go

```diff
@@ -542,36 +538,20 @@ func (s *RegionRequestSender) onSendFail(bo *Backoffer, ctx *RPCContext, err err
 	if span := opentracing.SpanFromContext(bo.GetCtx()); span != nil && span.Tracer() != nil {
 		span1 := span.Tracer().StartSpan("regionRequest.onSendFail", opentracing.ChildOf(span.Context()))
 		defer span1.Finish()
-		// TODO(MyonKeminta): Make sure trace works without cloning the backoffer.
-		// bo = bo.Clone()
 		bo.SetCtx(opentracing.ContextWithSpan(bo.GetCtx(), span1))
 	}
-	// If it failed because the context is cancelled by ourself, don't retry.
-	if errors.Cause(err) == context.Canceled {
-		return errors.Trace(err)
-	} else if atomic.LoadUint32(&ShuttingDown) > 0 {
+	if atomic.LoadUint32(&ShuttingDown) > 0 {
 		return tikverr.ErrTiDBShuttingDown
 	}
-	if status.Code(errors.Cause(err)) == codes.Canceled {
-		select {
```

Comment:
Why is this part removed? I didn't understand 😕

Reply:
Do you mean to keep the warning log?

Reply:
Ah, sorry, I think I got it. Nevermind.

Reply:
About the network error, I added a `TODO` and I will handle it carefully later.